### PR TITLE
Replace BOLT_* with GRANICA_*

### PR DESCRIPTION
**Pull request by gdaggarwal**

### CODE REVIEW

File: granica/__init__.py [hunk 1]
```diff
@@ -33,17 +33,17 @@ def __init__(self):
...
-        region = _environ.get('BOLT_REGION')
+        region = _environ.get('GRANICA_REGION')
         if region is None:
             try:
                 region = get_region()
             except Exception as e:
                 pass
         custom_domain = _environ.get('GRANICA_CUSTOM_DOMAIN')
         if custom_domain is None:
-            custom_domain = _environ.get('BOLT_CUSTOM_DOMAIN')
```
Comment (jr-projectn) [inline granica/__init__.py:44]: Hmm these changes aren't really needed, because I couldn't get these code paths work

File: granica/__init__.py [hunk 2]
```diff
@@ -33,17 +33,17 @@ def __init__(self):
...
-        service_url = _environ.get('BOLT_URL')
-        bolt_hostname = _environ.get('BOLT_HOSTNAME')
```
Comment (jr-projectn) [inline granica/__init__.py:46]: I believe these code paths don't work anymore. I removed them from the ReADME

Comment (gdaggarwal) [reply granica/__init__.py:46]: Should I keep it for now. 

If I remove them, how do I test. Is below test enough 

```python
import granica as boto3

s3_client = boto3.client('s3')
response = s3_client.get_object(Bucket='MyBucket', Key='MyKey.csv')
obj = response['Body'].read()

```

Comment (gdaggarwal) [reply granica/__init__.py:46]: Synced in person

**Review by cv-projectn — APPROVED** [review]

**Comment by gdaggarwal** [comment]
Not needed.

**Pull request closed without merging**